types/npmlog/tslint.json

```diff
@@ -1 +1,11 @@
-{ "extends": "dtslint/dt.json" }
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
```

Comment:
Please remove the rules here and fix errors. We want adherence to lint rules.

Reply:
@sheetalkamat Can you advise how I can adhere to this rule in npmlog-tests.ts? I can't figure out the syntax to combine those into a single import.